src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressioncommentfilter/InputSuppressionCommentFilter10.java

```diff
@@ -27,11 +18,6 @@
 applyToPrivate = (default)true
 
 
-com.puppycrawl.tools.checkstyle.checks.coding.IllegalCatch
-illegalClassNames = (default)Error, Exception, RuntimeException, Throwable, java.lang.Error, \
-                    java.lang.Exception, java.lang.RuntimeException, java.lang.Throwable
-
-
```

Comment:
whole content of this file is not relevant, please remove header comment, and whole body of class.
there should be only empty class.
As we do not use violation of this class.

Apply the same to another Input file.

Reply:
I removed all the files except `Filter config` as, without it, the exception is expected and the exception wasn't getting caught.

Reply:
please investigate a problem.
you use `execute(treeWalkerConfig` so it means that you get config created in test method (not a Input file).
Execute method does not parse Input file.

Reply:
I tried to remove all the comments from the filter config and found that:
```
/*
SuppressionCommentFilter
offCommentFormat = (default)CHECKSTYLE:OFF
onCommentFormat = (default)CHECKSTYLE:ON


*/
```

I removed other inputs and tests were passing but after removing  `offCommentFormat = (default)CHECKSTYLE:OFF` and  `onCommentFormat = (default)CHECKSTYLE:ON` the test gets fail and ask for exception is expected.

Reply:
I updated code to make it work, you need debug code.

Reply:
Changes Done. I got it.